### DISCUSSION TITLE
adding elk-hole to my security onion setup

### DISCUSSION THREAD

**Discussion by Magister-Machinis** [Security-Onion-Solutions/securityonion#2804] [Unsupported Versions]
Hello all,

I was wondering if anyone has tried their hand at adding elk-hole (repo below) to their security onion setups? I have a few piholes in my environment and this looks like it would be a handy thing to have. I'm a bit of an elk novice and I'm having difficulty translating the install instructions for elk-hole to the security onion docker/salt setup. Any help or advice would be appreciated.

https://github.com/nin9s/elk-hole

Thanks!

**Comment by weslambert** [comment]
I'd be careful with some of the paths and config.  We use ingest node parsing for most things in this version of Security Onion, so you would need to add additional LS config files to the pipeline(s), or create your own ingest pipelines with a converted config (from LS -> Ingest). 

https://docs.securityonion.net/en/latest/elasticsearch.html?highlight=parsing#parsing

**Reply by Magister-Machinis** [reply]
Thanks for the advice. Is it a matter of placing the elk-hole files in the security onion specific folders rather than vanilla locations? or do you believe the files need to be modified?

Thanks and be well.
